internal/ansible/events/log_events.go

```diff
@@ -62,22 +64,52 @@ func (l loggingEventHandler) Handle(ident string, u *unstructured.Unstructured,
 		"job", ident,
 	)
 
+	// Parse verbosity from CR
+	verbosityAnnotation := 0
+	if annot, exists := u.UnstructuredContent()["metadata"].(map[string]interface{})["annotations"]; exists {
+		if verbosityField, present := annot.(map[string]interface{})["ansible.sdk.operatorframework.io/verbosity"]; present {
```

Comment:
I can't remember if we need to do all the casting but it's probably the only way.

Reply:
same comment on unsafe casts here